pylint/checkers/utils.py

```diff
@@ -535,6 +536,104 @@ def next_char(i):
     return keys, num_args, key_types, pos_types
 
 
+if PY3K:
```

Comment:
Since we're now running only on Python 3, feel free to remove the `else` clause of this check.